lib/utils/format-time.js

```diff
@@ -46,21 +98,23 @@ function formatTime (epoch, translateTime = false) {
   }
 
   if (translateTime === true) {
-    return dateformat(instant, DATE_FORMAT_SIMPLE)
+    return simpleDateFormatter.format(instant)
```

Comment:
I'm confused as to why the `dateformat` function doesn't support the original format strings. We're having to do more work specific to date formatting in this module, which I don't think is a primary concern for it, with these changes.

Reply:
I certainly am not making that claim. I am trying to get the module incorporated. To do so, I want the incorporation to consist of replacing the module name in the `require` specifier only. I think this would be expected by anyone attempting to replace their usage of the original `dateformat` with the new one.